docs/guides/0x-animate-your-visualizations.md

```diff
@@ -0,0 +1,391 @@
+
+#### All together
+
+Well done! In the example below you will find the full code, including with the animation controls. You can play with it and make adjustments.
+
+Remember you can style your HTML controls using CSS. These styles are very simple, but at the beginning of this guide you have a complete example that includes beautiful styles. This example is also available in the [examples section](http://carto.com/developers/carto-vl/examples/#example-animation-controls).
+
+<div class="example-map">
+    <iframe
+        id="animation-step-5"
+        src="/developers/carto-vl/examples/maps/guides/animation/step-5.html"
+        width="100%"
+        height="500"
+        frameBorder="0">
+    </iframe>
+</div>
+
+```js
+const map = new mapboxgl.Map({
+    container: 'map',
+    style: carto.basemaps.darkmatter,
+    center: [-8.291255, 33.684553],
+    zoom: 2.8,
+    scrollZoom: false
+});
+
+map.touchZoomRotate.disableRotation();
+
+const nav = new mapboxgl.NavigationControl({
+    showCompass: false
+});
+
+map.addControl(nav, 'top-left');
+
+// Autenticate the client
+carto.setDefaultAuth({
+    username: 'cartovl',
+    apiKey: 'default_public'
+});
+
+// Create the source
+const source = new carto.source.Dataset('bird_journey');
+
+// Add better styles
+const viz = new carto.Viz(`
+    @duration: 30
+    @animation: animation($date_time, @duration, fade(1, 1))
+    filter: @animation
+    width: 4
+    strokeWidth: 0
+    color: #1785FB
+`);
+
+// Create the layer
+const layer = new carto.Layer('layer', source, viz);
+
+// Get animation control elements
+const $progressRange = document.getElementById('js-progress-range');
+const $playButton = document.getElementById('js-play-button');
+const $pauseButton = document.getElementById('js-pause-button');
+const $durationRange = document.getElementById('js-duration-range');
+
+// Listen to interaction events
+$playButton.addEventListener('click', () => {
+    viz.variables.animation.play();
+});
+
+$pauseButton.addEventListener('click', () => {
+    viz.variables.animation.pause();
+});
+
+$durationRange.addEventListener('change', () => {
+    viz.variables.duration = parseInt($durationRange.value, 10);
+});
+
+// Update progress each 100 milliseconds
+function updateProgress () {
+    $progressRange.value = viz.variables.animation.getProgressPct();
+}
+
+setInterval(updateProgress, 100);
+
+// Add the layer to the map
+layer.addTo(map);
+```
+
+### Advanced animations
+
+In this section, you will learn other properties that can be used to animate the visualization. You can change the input range of values by using a `linear` expression, and thus visualize a subset of the data. For example, if a dataset spans an entire year, but you only want to animate between the months of February and June, you can adjust the input parameter to:
+
+```js
+const viz = new carto.Viz(`
+    animation(linear($month, 2, 6))
```

Comment:
I would explain somehow that this is thx to the dataset having a $month column. Cause somebody could think this is done automagically!